spec/watirspec/radio_set_spec.rb

```diff
@@ -61,6 +61,7 @@
 
   context 'without name specified' do
     it 'Finds specified radio' do
+      browser.link
```

Comment:
Seems like a strange thing to add?

Reply:
I... still don't know what this is from. I need to investigate. :)

Reply:
hah. That was me committing a file that I didn't mean to commit. All fixed now.